react/features/jwt/reducer.js

```diff
@@ -0,0 +1,20 @@
+import { ReducerRegistry } from '../base/redux';
+
+import { SET_TOKEN_DATA } from './actionTypes';
+
+/**
+ * Listen for actions that update the state of token data part of the Redux
+ * store.
+ *
+ * @param {Object} state - Current state.
+ * @param {Object} action - Action object.
+ * @returns {Object}
+ */
+ReducerRegistry.register('features/jwt', (state = {}, action) => {
+    switch (action.type) {
+    case SET_TOKEN_DATA:
+        return action.tokenData;
```

Comment:
Which then makes me think the rest of the code is now incorrect because you can't destructure the state just like that, you need to destructure tokenData.